### PR TITLE
Update package.json

### LINKED ISSUE #62: Add support info
Ref: https://github.com/nodejs/package-maintenance/issues/413

Should also set the `engines` field.

### PR DESCRIPTION
**Pull request by rodion-arr**
- Adds support info
- Adds `engines` field

Closes #62

### CODE REVIEW

**Comment by rodion-arr** [comment]
I'm not fully confident in `engines` field, so if I've put not correct value, will be happy to update.

Based on CI - we run on LTS + v15. But I'm not sure how `engine` field like `"^15 || ^14 || ^13 || ^12 || ^10"` will behave in case this package will be installed on non-LTS, so I just put v10+.

**Comment by ljharb** [comment]
It should behave fine. If we don’t want to support node 11 (or 13, which also isn’t lts) we need the ^ form.

**Comment by rodion-arr** [comment]
Thanks, updated

**Review by ljharb — APPROVED** [review]

**Review by BethGriggs — APPROVED** [review]

**Comment by dominykas** [comment]
Thanks for this PR!

I'm not sure we should "support" v15 - we should use `lts_active` policy instead and remove the v15 from the engines. Testing in v15 is just to get a heads up before the next LTS.

File: package-support.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "versions": [
+    {
+      "version": "*",
+      "target": {
+        "node": "active"
```
Comment (dominykas) [inline package-support.json:6]:
```suggestion
        "node": "lts_active"
```

**Review by dominykas — APPROVED** [review]

**Pull request merged**